### ISSUE TITLE
Mouse not default enabled when starting Modmanager.

### ISSUE THREAD

**Issue by liesemans** [LeeTwentyThree/SubnauticaModManager#1]
When I start the modmanager in subnautica my mouse control seems to be disabled,  only after going in game, then back to menu and again activating the mod manager the mouse can be controlled.

**Comment by LeeTwentyThree** [comment]
So you have mouse control in the main menu but you lose it as soon as you enter the mod manager? That’s strange, I don’t believe anything remotely related to that is written anywhere nor have I heard it before. What mods do you have?

**Comment by LeeTwentyThree** [comment]
Can you please test this and see if it fixes your problem? https://github.com/LeeTwentyThree/SubnauticaModManager/releases/tag/1.0.4

**Comment by liesemans** [comment]
Hi there, thanks for your quick responses, much appreciated.
I tested 1.0.4, the UI scaling is OK now!

Regarding mouse control, let me try to explain what happens:
1. I start Subnautica, and the BepInex console is popping up with the messages.
2. As soon as this happens and the Subnautica splashscreen is loaded my mousepointer is locked to the middle position of the display and can't be moved with the mouse.
3. When I ALT-TAB a few times the mouse is released and I can bring the focus to the Subnautica screen.
4. When Subnautica is ready loading I chose MODMANAGER from the main menu, modmanager loads OK.
5. I can control the mouse pointer in the modmanager but I can't select anything, buttons do not work.
6. I EXIT the Modmanager with ESCAPE to return to the main menu and select PLAY.
7. Once in the game I exit the game to the main menu and select Modmanager again and this time everything seems to be normal and all functions of modmanager are working.
The above procedure is reproducable each time.
The only mod I loaded is the "More Quick Slots mod v1.2.0)

Hope thjs helps.

**Comment by LeeTwentyThree** [comment]
What system are you running the game on? What launcher are you using (Steam, Epic, Windows Store, etc...)? And where did you get your version of BepInEx from? For some reason, I can't even get the BepInEx console to appear in the first place.

**Comment by LeeTwentyThree** [comment]
Solution for this issue appears to be:
Reinstall BepInEx, and reinstall Subnautica if that does not work

Your new issue seems to be the same as [this](https://github.com/LeeTwentyThree/SubnauticaModManager/issues/3). Closing this one, new conversation should take place there.
